src/rebar_prv_shell.erl

```diff
@@ -86,6 +89,7 @@ shell(State) ->
     setup_name(State),
     setup_paths(State),
     setup_shell(),
+    maybe_run_script(State),
```

Comment:
I think the script stuff should run before booting Apps - I'm intending that people use it to setup fake servers that they would connect to on boot anyway. It would be nice to get access to their OTP config, but that would require splitting load and start as you say. I'm going to try the easier thing for now.